index.src.html

```diff
@@ -1903,6 +1921,11 @@ <h4 algorithm id="construct-federatedcredential-data">
       {{Credential/[[DiscoverFromExternalSource]](options)}} is appropriate for
       [=credentials=] that need to be re-generated from a [=credential source=].
 
+      Long-running operations, like those in {{PublicKeyCredential}}'s
+      {{PublicKeyCredential/[[Create]](options)}} and {{PublicKeyCredential/[[DiscoverFromExternalSource]](options)}}
```

Comment:
Nit: This causes Bikeshed to pull WebAuthn in as a normative reference, which I don't think we actually want. That said, I'm not sure how to tell Bikeshed that it's an informative reference. :) @tabatkins might have some ideas? :)

Reply:
Whoops. https://tabatkins.github.io/bikeshed/#auto-biblio says how to avoid it. Done.